lib/elixir/lib/code.ex

```diff
@@ -50,9 +50,11 @@ defmodule Code do
 
   ## Examples
 
-      Code.append_path(".") #=> true
+      iex> Code.append_path(".")
```

Comment:
right. I'll just revert changes in code.ex except for `available_compiler_options` which is pure